org.mwc.debrief.track_shift/src/org/mwc/debrief/track_shift/ambiguity/AmbiguityResolver.java

```diff
@@ -1462,7 +1462,7 @@ else if (valueDelta > 180)
                 + " lastBrg:" + (int) lastCut.getBearing() + " brg:"
                 + ((int) cut.getBearing()) + " brg delta:" + brgRate
                 + " gap rate:" + gapRate + " OS zig trip:" + TRIP_ZIG;
-            doLog(logger, stats, time);
```

Comment:
@saulhidalgoaular - in this file, a _slightly_ less intrusive change would be to just comment out the contents of `doLog()`.